Bytesize transcript gathertown (#1511)
nf-core/nf-co.re: markdown/events/2022/bytesize-37-gathertown.md

[7:35](https://www.youtube.com/watch?v=nRY1_FvL7Pk&t=455)
Here we have our first classroom. This will be for people who are completely new to Nextflow and completely new to nf-core. It's a space to do self-learning together, follow tutorials that exist already either Seqera or nf-core. And you can go step by step together and do this group learning exercise there. Another nice thing is that in most rooms, we have these whiteboards, which if you go up and press X, it should load something like this. This is a cross-collaborative whiteboarding software which Gather Town offers. You can write things here, but you can also, collaboratively write things, draw pictures and images or draw diagrams if you need to help each other work out what's going on. These should saved and be persistent between usage. There you go, you can see on the left, we've got a graffiti artist going on there. We'll leave that. That's the classroom specifically for people who are completely new and want to just get started with Nextflow or nf-core.

[8:39](https://www.youtube.com/watch?v=nRY1_FvL7Pk&t=519)

> But for the main topic of the hackathon, we will have four different groups and each different group will be assigned to one of these rooms. If we go here, you see it's basically the same thing. It's a bigger space, we can have more people, but you still have the whiteboards. The nice thing about the tables is if we go into the tables, anyone sitting around the table can hear each other, but when you're moving away, well, that's not working here, I need to check that, but basically everyone sitting around the table will be isolated with themselves. People standing away from the table will not be able to hear the conversations in the board. That means you can also have everybody still in the workspace working on their own things, we can have teams who are not interfering with each other, which is a really nice feature.

Reviewer: ```suggestion
But for the main topic of the hackathon, we will have four different groups and each different group will be assigned to one of these rooms. If we go here, you see it's basically the same thing. It's a bigger space, we can have more people, but you still have the whiteboards. The nice thing about the tables is if we go into the tables, anyone sitting around the table can hear each other, but when you're moving away, well, that's not working here, I need to check that, but basically, everyone sitting around the table will be isolated with themselves. People standing away from the table will not be able to hear the conversations in the board. That means you can also have everybody still in the workspace working on their own things, we can have teams who are not interfering with each other, which is a really nice feature.
```

[9:24](https://www.youtube.com/watch?v=nRY1_FvL7Pk&t=564)

> I can maybe show you that back in the lobby if Maxime comes with me. Or, okay, this should work, I'm not sure if it's working here, but theoretically it'll work. (Maxime: I think we just forgot to put some private spaces in the room. I'll do that later.) Yeah, good. Then in addition to the classrooms or group rooms, our main room for all the check-ins, check-outs, talks, presentations is here is the lecture theatre. you can see at the top here, there's Matthias already there giving us a dance recital, very kind of him, and Maxime as well. The important thing here is that anyone standing on those orange dots by the pedestals are projecting to the whole room. You can see that Matthias, for example, has this little orange icon on, Maxime as well, saying that everybody in that room can hear him. This overrides this proximity communication bit.

Reviewer: ```suggestion
I can maybe show you that back in the lobby if Maxime comes with me. Or, okay, this should work, I'm not sure if it's not working here, but theoretically, it'll work. (Maxime: I think we just forgot to put some private spaces in the room. I'll do that later.) Yeah, good. Then in addition to the classrooms or group rooms, our main room for all the check-ins, check-outs, talks, and presentations is here in the lecture theatre. You can see at the top here, there's Matthias already there giving us a dance recital, very kind of him, and Maxime as well. The important thing here is that anyone standing on those orange dots by the pedestals is projecting to the whole room. You can see that Matthias, for example, has this little orange icon on, Maxime as well, saying that everybody in that room can hear him. This overrides this proximity communication bit.
```

[10:22](https://www.youtube.com/watch?v=nRY1_FvL7Pk&t=622)
On the right hand side, we have the second... the one on the main platform is for the speakers. If you ever have any questions, you can go and line up over here, exactly, nice queue. Whoever's standing on this orange bit will also be projected the room. if you have any questions during the talk, you can come up here to ask your questions. Now, the final room we can go into is where we have all the social activities, this is here in the nf-core cafe. Here's a good example of the private space around all the tables. We'll make sure this is everywhere. Anyone not in this square of the green square there cannot hear us. But if Maxime comes in, now he comes in, he can now join the conversation. And so, you have these private conversations as you want.

[11:15](https://www.youtube.com/watch?v=nRY1_FvL7Pk&t=675)

> Other things about the user interface, there is also text chat if necessary. You can see this here. If you ever need to refer to the code of conduct, that's at the top here. You can change between different ways of chatting with people, either private chat to everybody in the entire space, like the entire world, and also any people in your close proximity. Another thing you can do is let's say you are lost. There's a little mini map down here, which can tell you where you are, and it'll point out where anybody else is. particularly in bigger spaces, if you can't find someone, you can look here. You can also search for people or follow people if necessary. Let's say, you know, I want to meet Matthias, I can press follow here. Automatically, my little avatar will go and walk towards him in the most fast route possible, then we can run. Also I can keep following him if he wants to lead me somewhere. If you're a bit too nervous and get stuck or lost, you can just follow people, that speeds things up as well. If at any point you get stuck, you can also... let's say, I come here, I can't walk through Matthias, but if I need to walk through Matthias because it's too big of a crowd, if I hold down G, I can then walk through the person. I turn them into a ghost, so G for ghost, and you can get through them. If you ever get stuck, that normally will help you with this.

Reviewer: ```suggestion
Other things about the user interface, there is also text chat if necessary. You can see this here. If you ever need to refer to the code of conduct, that's at the top here. You can change between different ways of chatting with people, either private chat to everybody in the entire space, like the entire world and also any people in your close proximity. Another thing you can do is let's say you are lost. There's a little mini-map down here, which can tell you where you are, and it'll point out where anybody else is. particularly in bigger spaces, if you can't find someone, you can look here. You can also search for people or follow people if necessary. Let's say, you know, I want to meet Matthias, I can press follow here. Automatically, my little avatar will go and walk towards him in the fastest route possible, then we can run. Also, I can keep following him if he wants to lead me somewhere. If you're a bit too nervous and get stuck or lost, you can just follow people, that speeds things up as well. If at any point you get stuck, you can also... let's say, I come here, I can't walk through Matthias, but if I need to walk through Matthias because it's too big of a crowd, if I hold down G, I can then walk through the person. I turn them into a ghost, so G for ghost, and you can get through them. If you ever get stuck, that normally will help you with this.
```
